### ISSUE TITLE
Renovate from mend https://www.mend.io/free-developer-tools/renovate/

### ISSUE THREAD

**Issue by ncsokas** [ncsokas/github-actions-training#35]
> Save time and reduce risk by automating dependency updates in software projects

Create a new card in the Backlog list with proper descriptions of the project

Fields to populate:
Description about the tool
Similar known artifacts:
Reason for assessing (What problem is it solving?):
Research tasks: (Examples: How is it maintained, price, OS?, where does it fit in the ecosystem, how will it be used?)
Prerequisite: What should be present before the spike
Starting litterature

┆Card is synchronized with this [Github issue](https://github.com/ncsokas/100-days-of-code/issues/68) by [Unito](https://www.unito.io)
